cypress/integration/customer_creation.js

```diff
@@ -0,0 +1,23 @@
+
+context('Customer', () => {
+	before(() => {
+		cy.login();
+	});
+	it('Insert a Customer ', () => {
+		cy.visit(`app/customer/`);
+		cy.get('.primary-action').click();
+		cy.get('.custom-actions > .btn').click();
+		cy.get_field('customer_group', 'Link').type("All Customer Groups");
+		cy.get_field('customer_name', 'Link').type("Lara Schindler");
+		cy.get_field('territory', 'Link').type("All Territories");
+		cy.get('#page-Customer > .page-head > .container > .row > .col > .standard-actions > .primary-action').click();
```

Comment:
If this is "clicking save" then it's better to create a custom command for this. There are 4 different selectors for doing "click save" in this PR. 

many examples here: https://github.com/frappe/frappe/blob/develop/cypress/support/commands.js